administrator/language/de-DE/com_installer.ini

```
@@ -195,7 +195,7 @@ COM_INSTALLER_MSG_WARNINGS_NOTCOMPLETE="<h1>Warnung: Update nicht vollständig!<
 COM_INSTALLER_MSG_WARNINGS_PHPUPLOADNOTSET="Das Verzeichnis für temporäre Dateien in PHP ist nicht gesetzt"
 COM_INSTALLER_MSG_WARNINGS_PHPUPLOADNOTSETDESC="Dieses Verzeichnis wird zum temporären Speichern von hochgeladenen Dateien benutzt, bevor Joomla! auf die Datei zugreifen kann. Obwohl das temporäre Verzeichnis nicht gesetzt wurde, sollte es in der Regel keine Probleme geben. Wenn es Probleme gibt, dass XML-Dateien (Manifest-Dateien) nicht erkannt oder hochgeladene Dateien nicht gefunden werden, sollte der Wert von <code>upload_tmp_dir</code> in der „php.ini“-Datei angepasst werden."
 COM_INSTALLER_MSG_WARNINGS_PHPUPLOADNOTWRITEABLE="Das Verzeichnis für temporäre Dateien in PHP ist schreibgeschützt."
-COM_INSTALLER_MSG_WARNINGS_PHPUPLOADNOTWRITEABLEDESC="Das in der „php.ini“-Datei eingestellte temporäre Verzeichnis <code>upload_tmp_dir</code> ist schreibgeschützt und kann daher von Joomla! nicht genutzt werden. Dies kann zu Problemen beim Hochladen von Erweiterungen in Joomla! führen. Wenn es Probleme beim Hochladen von Erweiterungen gibt, sollte das Hochladeverzeichnis („%s“) überprüft werden und ggf. der Schreibschutz aufgehoben werden."
+COM_INSTALLER_MSG_WARNINGS_PHPUPLOADNOTWRITEABLEDESC="Das in der „php.ini“-Datei eingestellte temporäre Verzeichnis <code>upload_tmp_dir</code> ist schreibgeschützt und kann daher von Joomla nicht genutzt werden. Dies kann zu Problemen beim Hochladen von Erweiterungen in Joomla führen. Wenn es Probleme beim Hochladen von Erweiterungen gibt, sollte das Hochladeverzeichnis („%s“) überprüft werden und ggf. der Schreibschutz aufgehoben werden."
```

```suggestion
COM_INSTALLER_MSG_WARNINGS_PHPUPLOADNOTWRITEABLEDESC="Das in der „php.ini“-Datei eingestellte temporäre Verzeichnis <code>upload_tmp_dir</code> ist schreibgeschützt und kann daher von Joomla! nicht genutzt werden. Dies kann zu Problemen beim Hochladen von Erweiterungen in Joomla! führen. Wenn es Probleme beim Hochladen von Erweiterungen gibt, sollte das Hochladeverzeichnis („%s“) überprüft werden und ggf. der Schreibschutz aufgehoben werden."
```
see Comment on line R97 file administrator/language/de-DE/com_config.ini